examples/beam/default.story.vue

```diff
@@ -62,6 +104,26 @@ const workOrder = reactive({
 	complete: false,
 })
 
+// Toast Component //
+
+const toast = useToast()
+const toastType = ref('default')
+const toastTime = ref(3)
+const toastMsg = ref('Toast Message.')
+const toastPosition = ref('top')
+
+const showNotification = () => {
+	let toastOptions = {
+		message: toastMsg.value,
+		position: toastPosition.value,
+		duration: toastTime.value * 1000,
+		type: toastType.value,
+	}
+	// let notification = toast.success("Notified!", toastOptions)
+	let notification = toast.open(toastOptions)
+}
```

Comment:
@crabinak @agritheory Should we create a wrapper around `vue-toast-notification` so that users can import it from `@stonecrop/beam` rather than having to remember to use the third-party library? Maybe we package the custom styles into our composable too?

Reply:
@Alchez @crabinak I agree. Curt is working on the style overrides but I don't think they're included here. There's a likely case where we want to use a toast notification in a desktop/aform type environment too,

Reply:
@agritheory @crabinak

Okay, so I tried to create a composable wrapper around the dependency but there seems to be an issue with it that I can't locate the source of.

- If I keep the dependency in `examples`, then I get errors during the build but toasts still work inside the story so the errors are non-blocking.
  - Someone already raised an [issue](https://github.com/ankurk91/vue-toast-notification/issues/97) last month for the errors I saw but the author isn't interested in maintaining the project anymore. 
- If I move the dependency to `beam` and try to create a re-usable composable, then the build fails entirely and I'm unable to access any of the Beam stories. I'm not sure what's going on there but this is blocking.

I think we'll have to either:
- look at another dependency
- somehow fix the existing dependency
- or recommend using the library manually in the target app (which kinda defeats the purpose of this PR)

What do you guys think?